src/components/maincontroller.ts

```diff
@@ -372,40 +373,45 @@ export function setSubtitleStreamIndex($scope, index) {
     }
 }
 
-export function setAudioStreamIndex($scope, index) {
-    var positionTicks = getCurrentPositionTicks($scope);
-    changeStream(positionTicks, {
+export function setAudioStreamIndex(
+    $scope: GlobalScope,
+    index: number
+): Promise<void> {
```

Comment:
Same as above

Reply:
I'm not sure but I think I put this one to void now